config/locales/en.yml

```diff
@@ -1,6 +1,6 @@
 en:
   field_text_block_text: Text
-  label_select_text_block: "Select text block..."
+  label_select_text_block: "Select text block…"
```

Comment:
@dkastl I searched multi-byte `…` is used in original Redmine `config/*.yml`, but it only used in `gl.yml` and `sq.yml`.
So, I think that it's better to use 3 dots on `de.yml`, `en.yml` and `ja.yml`.
![multibyte-3-dots-in-redmine-config-locales](https://user-images.githubusercontent.com/629923/189589851-e1cc0da8-5bec-4d9d-b565-3f62349af155.png)

Reply:
Okay.
If Weblate setting accepts a customization of such suggestions, it will be nice.

Reply:
@dkastl  
Okay. I checked both English and Japanese, and the view looks seem to be almost same, so using multi-byte `…` in all languages may be also fine.

I confirmed that QGIS also uses multi-byte '…', so let's use this one, even if Redmine upstream still keeps 3 dots '...'.